File: Week02/Day3/match_making.py
# Join the two lists by matching one girl with one boy in the order list
# Exepected output: ["Eve", "Joe", "Ashley", "Fred"...]

girls = ["Eve", "Ashley", "Bozsi", "Kat", "Jane"]
boys = ["Joe", "Fred", "Bela", "Todd", "Neef", "Jeff"]

def matching(girls, boys):
    order = boys
    a = 0
    for i in (girls):
            order.insert (a,i)
            a += 2
    print (order)
    return (order)

matching(girls,boys)
  

File: Week07/Day3/practice/matchmaking.js
'use strict';
// Join the two array by matching one girl with one boy in the order array
// Exepected output: ["Eve", "Joe", "Ashley", "Fred"...]

var girls = ["Eve","Ashley","Bözsi","Kat","Jane"];
var boys = ["Joe","Fred","Béla","Todd","Neef","Jeff"];
var order = [];

// for (let i = 0; i < girls.length; i++) {
//     for (let j = 0; j < boys.length; j++) {
//         if (i === j) {
//             order.push(girls[i]);
//             order.push(boys[j])
//             // + Jeff
//         }
//     }
// }

// console.log(order);

order = boys
for (let i = 0; i < girls.length; i++) {
        order.splice(i+2, 0, girls[i]);
            }

console.log(order);

